### PR TITLE
pretty flow

### PR DESCRIPTION
**Pull request by thelostone-mc**

### CODE REVIEW

File: src/utils/config.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const aspectRatio = 7 / 1;
+export const aspectRatio = 7 / 2;
```
Comment (thelostone-mc) [inline src/utils/config.ts:1]: ^ need to see if this is better size for all screen sizes

File: src/components/application/ApplicationForm.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const schema = yup.object({
   website: yup.string().required().url("Must be a valid website address"),
   description: yup.string().required().min(10, "Must be at least 150 words"),
   email: yup.string().required().min(3).email("Must be a valid email address"),
-  requestedAmount: yup.number().required("Requested amount is required"),
+  // requestedAmount: yup.number().required("Requested amount is required"),
```
Comment (thelostone-mc) [inline src/components/application/ApplicationForm.tsx:20]: This keeps failing and i'm not sure why.

File: src/components/application/ApplicationForm.tsx [hunk 3]
```diff
@@ -194,16 +194,16 @@ export default function ApplicationForm() {
 
             <div className="sm:col-span-4">
               <label
-                htmlFor="requested-amount"
+                htmlFor="requestedAmount"
```
Comment (thelostone-mc) [inline src/components/application/ApplicationForm.tsx:197]: ^ This still fails. Need to figure out why

File: src/utils/config.ts (hunk 1 above)
Comment (0xKurt) [reply src/utils/config.ts:1]: looks better

**Pull request merged**